code/modules/shuttle/emergency.dm

```diff
@@ -175,8 +180,26 @@
 	if(ENGINES_STARTED || (!IS_DOCKED))
 		return .
 
+
+	// monkestation addition
+	if(obj_flags & EMAGGED && (!emaglast || emagcooldown + emaglast < world.time) && prob(50))
+		var/cut_time = 2 / rand(1, 5)
+
+		if (cut_time * TIME_LEFT <= 11)
+			SSshuttle.emergency.setTimer(ENGINES_START_TIME)  // to make it align with igniting
+		else
+			SSshuttle.emergency.modTimer(cut_time)
+			if (cut_time > 1)  // providing launch updates if it changes (doesnt roll a 2)
```

Comment:
Since 2 / 2 would be 0, you can have this check for any value.
```suggestion
			if (cut_time)  // providing launch updates if it changes (doesnt roll a 2)
```

Reply:
i thought 2/2 was 1?